CONTRIBUTING.md

```diff
@@ -32,6 +32,13 @@ Pull requests are also welcome on GitHub: https://github.com/puppetlabs/bolt
 As with other open-source projects managed by Puppet, you must digitally sign the Contributor
 License Agreement before we can accept your pull request: https://cla.puppet.com
 
+To submit a PR:
+1. Fork the Bolt project (button in the top-right, next to 'star' and 'watch')
+2. Clone your fork of Bolt
+3. Make a new branch off of master - `git checkout -b mybranchname`
```

Comment:
Is adding the upstream repo too advanced for this? This works fine for the case where someone wants to make a quick one-off change, but if they come back to it later their fork will have gotten out of date.

Reply:
Great point! Updated